src/components/editor/property/LanguageButton.jsx

```diff
@@ -70,7 +70,7 @@ const mapStateToProps = (state, ourProps) => {
   const node = findNode(state.selectorReducer, ourProps.reduxPath)
   const item = node.items.find(item => item.id === ourProps.id)
   return {
-    language: languageLabel(state, item.lang.id),
+    language: languageLabel(state, item.lang),
```

Comment:
is this function name still appropriate?

Reply:
yes, this is looking up the language label from the entity store.